lib/srv/reexec.go

```diff
@@ -608,6 +610,105 @@ func runPark() (errw io.Writer, code int, err error) {
 	}
 }
 
+const (
+	waitNoResolveDefaultPeriod  = 10 * time.Second
+	waitNoResolveDefaultTimeout = 10 * time.Minute
+)
+
+func waitNoResolve() (io.Writer, int, error) {
```

Comment:
I don't know if this belongs in lib/srv/reexec to be honest. Why not just keep it in tool package e.g. `tool/teleport/common/wait.go` or something?

Reply:
Done